### DISCUSSION TITLE
Add support for Mini games highscores etc over network

### DISCUSSION THREAD

**Discussion by Zedb0T** [open-goal/jak-project#3054] [Jak 2]
Since the addition of SRC leaderboards support it would be cool to add a way for the game to automatically submit/store/validate minigame highscores as well and display them in the menu

https://opengoal.dev/blog/progress-report-sept-2023/#integrate-speedruncom-leaderboards-in-high-score-menu-

**Comment by Zedb0T** [comment]
Additionally I think it would be cool but maybe a bit more work to automatically submit segment times for certain missions etc...

**Reply by xTVaser** [reply]
The issue is you need a reliable source of truth, highscores will never work on speedrun.com.  So it's something that I'm hoping we can do on jakspeedruns.com.  So the problem is out of the scope for `jak-project` right now.

As for automatic submission of stuff, while I do want to work on some practice tooling, there are better ways to submit segment times (livesplit and splits.io).
